frontend/pages/policies/PolicyPage/components/QueryResults/QueryResults.tsx

```diff
@@ -135,8 +136,30 @@ const QueryResults = ({
       </div>
     );
   };
+  const renderPassFailPcts = () => (
+    <span className={`${baseClass}__results-pass-fail-pct`}>
+      {" "}
+      (Yes:{" "}
+      <TooltipWrapper
+        tipContent={`${hostsCount.successful} host${
```

Comment:
Ahh yeah, hostsCount.successful is all successful results, passing or failing

Reply:
Looks like you're going to have to go through the host returned and keep track of the count in state

<img width="1341" alt="Screenshot 2024-04-15 at 2 18 10 PM" src="https://github.com/fleetdm/fleet/assets/71795832/5aff5aaa-048f-4788-9941-8a52f50efefc">